backend/py_neuro/main.py

```
@@ -1,47 +1,60 @@
 from fastapi import FastAPI
-from fastapi.middleware.cors import CORSMiddleware # Import CORS middleware
-# Use absolute imports relative to py_neuro
-from py_neuro.routers import knowledge_graph, topics, study_guide, workspaces, files # Add files router
-from py_neuro.database import connect_to_mongo, close_mongo_connection
+from fastapi.middleware.cors import CORSMiddleware
+from .routers import knowledge_graph, topics, study_guide, workspaces, files
```

Ruff (0.8.2)

`.routers.study_guide` imported but unused

Remove unused import: `.routers.study_guide`

(F401)